documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorage.java

```diff
@@ -538,7 +538,7 @@ public void renameDocument(long ownerId, String documentID, String title, Identi
       } else {
         node = getNodeByIdentifier(session, documentID);
       }
-      String name = Text.escapeIllegalJcrChars(cleanString(title));
```

Comment:
we should not remove cleanString here, instead, we need just to clean the content name and preserve the title as is.